resources/profiles/AnkerMake.ini

```diff
@@ -286,14 +258,14 @@ filament_colour = #DDDDDD
 filament_type = ABS
 filament_density = 1.04
 filament_cost = 20
-first_layer_bed_temperature = 100
-first_layer_temperature = 245
+first_layer_bed_temperature = 90
+first_layer_temperature = 260
```

Comment:
@ankermake I would reduce this to 255 as you are pushing the limit of the PTFE tube for off-gassing since you do not have an all-metal hotend.